app/views/devise/mailer/confirmation_instructions.html.haml

```diff
@@ -2,5 +2,9 @@
   Welcome #{@email}!
 %p You have been invited to contribute to the Draw My Life admin portal, collecting drawings and data to support children in crisis.
 
-%p Please click the link below to confirm your Draw My Life  account email and set your password:
+%p
+  Please click the link below
+  - unless Devise.confirm_within.nil?
+    %b within #{time_ago_in_words(Devise.confirm_within.seconds.from_now)}
```

Comment:
As we both spotted IRL just then (but relaying here for anyone else) we can use `Devise.confirm` without formatting as seconds here and it should have the same result. Also, might want to use an alternative method to "wordify" the time as `time_ago_in_words` can have some fuzzy results, e.g.

```
[6] pry(main)> helper.time_ago_in_words(Devise.confirm_within.from_now)
=> "7 days"
[7] pry(main)> Devise.confirm_within = 14.months
=> 14 months
[8] pry(main)> helper.time_ago_in_words(Devise.confirm_within.from_now)
=> "about 1 year"
[9] pry(main)> helper.time_ago_in_words(Devise.confirm_within.seconds.from_now)
=> "about 1 year"
```